### PR TITLE
ADD:Config import 

### PR DESCRIPTION
**Pull request by MaxTheGeeek**
## Frontend
- [x]  Uploading the config file
- [x] Validating file format
- [x] List of imported config
- [x] Syncing imported config

---

## Backend

- [x] Get the yaml files from Frontend
- [x] check sync point. Put the new one if it already exists
- [x] check SSV. Create SSV config file if it exists
- [x] Prepare Node
- [x] Start imported services

---
https://github.com/stereum-dev/ethereum-node/issues/961

### CODE REVIEW

**Comment by MaxTheGeeek** [comment]
UI done.

File: launcher/src/components/UI/plugin-installation/ImportingVerify.vue [hunk 1]
```diff
@@ -68,44 +68,16 @@ export default {
...
+      this.configServices = this.configServices.map((service) => {
+        const volumesRegex = /\/opt\/\w+\//g;
+        let matches = service.content ? service.content.match(volumesRegex) : "";
+        matches = matches !== null ? matches.filter((i) => i !== "/opt/app/") : "";
 
-        const path = matches[0].trim();
-        console.log("firstPath: ", path);
+        return {
+          ...service,
+          configPath: matches[0] !== undefined ? matches[0] : "/opt/stereum/",
```
Comment (gbayasgalan) [inline launcher/src/components/UI/plugin-installation/ImportingVerify.vue:78]: IMHO default path cannot be `/opt/stereum`

File: launcher/package.json [hunk 2]
```diff
@@ -33,6 +33,7 @@
     "electron-store": "^8.1.0",
     "electron-updater": "^5.2.1",
     "file-saver": "^2.0.5",
+    "js-yaml": "^4.1.0",
```
Comment (NeoPlays) [inline launcher/package.json:36]: pls remove

Comment (NeoPlays) [inline launcher/package.json:36]: pls remove

**Review by NeoPlays — CHANGES_REQUESTED** [review]

File: launcher/src/backend/ServiceManager.js [hunk 3]
```diff
@@ -1106,4 +1106,36 @@ export class ServiceManager {
     }
     return serviceNameConfig;
   }
+
```
Comment (NeoPlays) [inline launcher/src/backend/ServiceManager.js:1109]: also dont forget this

**Review by NeoPlays — CHANGES_REQUESTED** [review]

File: launcher/src/backend/ServiceManager.js [hunk 4]
```diff
@@ -1106,4 +1106,36 @@ export class ServiceManager {
...
+    //         console.log(`child process exited with code ${code}`);
+    //       });
+    //     });
+
+    //     // Upload the file
+    //     // ...
+    //   });
+    // };
+    // reader.readAsArrayBuffer(file);
+  }
```
Comment (NeoPlays) [inline launcher/src/backend/ServiceManager.js:1140]: dont forget about this

**Review by NeoPlays — CHANGES_REQUESTED** [review]

File: launcher/src/backend/ServiceManager.js [hunk 5]
```diff
@@ -835,12 +800,12 @@ export class ServiceManager {
 
   async createKeystores(services) {
     for (const service of services) {
-      if (service.service === "NimbusValidatorService" || (service.service === "NimbusBeaconService" && service.configVersion < 2)) {
+      if (service.service.includes("Nimbus")) {
```
Comment (NeoPlays) [inline launcher/src/backend/ServiceManager.js:803]: guys if you have merge conflicts don't just push anything... if you're not sure, then ask the one who wrote this and don't just do something 😅

**Review by NeoPlays — CHANGES_REQUESTED** [review]

File: launcher/vue.config.js [hunk 6]
```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line no-unused-vars
 const path = require("path");
```
Comment (NeoPlays) [inline launcher/vue.config.js:2]: can we remove this?

**Review by NeoPlays — CHANGES_REQUESTED** [review]

File: launcher/src/backend/ServiceManager.js (hunk 5 above)
Comment (gbayasgalan) [reply launcher/src/backend/ServiceManager.js]: got it sir:)

File: launcher/vue.config.js (hunk 6 above)
Comment (gbayasgalan) [reply launcher/vue.config.js:2]: not sure, @MaxTheGeeek what do you think? probably need somewhere in frontend?

Comment (gbayasgalan) [reply launcher/vue.config.js:2]: tested, and `import-config` works fine without this. So I just remove it for now. :)

**Comment by MaxTheGeeek** [comment]
What's going on here? what u doin guys? :) 
I only get notifications :))

File: launcher/vue.config.js (hunk 6 above)
Comment (MaxTheGeeek) [reply launcher/vue.config.js]: > can we remove this?

I'd remove it a few times but thought maybe someone needs

File: launcher/package.json (hunk 2 above)
Comment (gbayasgalan) [reply launcher/package.json]: @MaxTheGeeek I think it is used to load .zip file right?

File: launcher/vue.config.js (hunk 6 above)
Comment (NeoPlays) [reply launcher/vue.config.js]: acutally i ment this:  `// eslint-disable-next-line no-unused-vars` 😅

Comment (gbayasgalan) [reply launcher/vue.config.js]: done!

Comment (MaxTheGeeek) [reply launcher/vue.config.js]: > acutally i ment this: `// eslint-disable-next-line no-unused-vars` 😅

That belonged to the path also.

**Review by NeoPlays — APPROVED** [review]
lgtm

**Pull request merged**